### DISCUSSION TITLE
AttributeError at /fobi/forms/elements/add/2/text/

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [barseghyanartur/django-fobi#283] [Q&A]
Creating a form in the admin ui seemed to be ok, but when clicking view it shows the form but no field elements.  From there clicking 'Edit' it shows no field elements, so I tried to add a text element and when I clicked 'ADD' I get the following...

<html>
<body>
<h1 style="scroll-behavior: auto !important; padding: 0px; margin: 0px; font-weight: normal; color: rgb(0, 0, 0); font-family: sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">AttributeError at /fobi/forms/elements/add/2/text/</h1><pre class="exception_value" style="scroll-behavior: auto !important; padding: 0px; margin: 10px 0px; font-size: 1.5em; white-space: pre-wrap; word-break: break-word; font-family: sans-serif; color: rgb(87, 87, 87); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">'FormElementPluginDataStorage' object has no attribute 'name'</pre>

Request Method: | POST
-- | --
http://localhost:8000/fobi/forms/elements/add/2/text/
4.1.1
AttributeError
'FormElementPluginDataStorage' object has no attribute 'name'
C:\Users\oliver\base\correlator2v2\correlator2home\venv\lib\site-packages\fobi\base.py, line 859, in validate_plugin_data
fobi.views.function_based.add_form_element_entry
C:\Users\oliver\base\correlator2v2\correlator2home\venv\scripts\python.exe
3.10.6
['C:\\Users\\oliver\\base\\correlator2v2\\correlator2home',  'C:\\Python3\\python310.zip',  'C:\\Python3\\DLLs',  'C:\\Python3\\lib',  'C:\\Python3',  'C:\\Users\\oliver\\base\\correlator2v2\\correlator2home\\venv',  'C:\\Users\\oliver\\base\\correlator2v2\\correlator2home\\venv\\lib\\site-packages']
Sat, 10 Sep 2022 08:32:02 +0000

</body>
</html>

So close, if somebody can point me to where I should be looking.

**Comment by barseghyanartur** [comment]
You're not supposed to create forms in Django admin. You should be using the django-fobi interface for it.

**Reply by michaeloliveraz** [reply]
Ok fine, how do I navigate there?  If not supposed to create forms in Django Admin then why is there an interface there to do that?  How do I add a form using the Fobi Management interface and how do I get there? Your help is appreciated. If you can point me to the documtation that I missed, that would help.
